src/Adapter/Cart/QueryHandler/GetCartInformationHandler.php

```diff
@@ -158,13 +158,6 @@ private function getAddresses(Cart $cart): array
 
         foreach ($customer->getAddresses($cart->id_lang) as $data) {
             $addressId = (int) $data['id_address'];
-            $countryIsEnabled = (bool) Address::isCountryActiveById($addressId);
```

Comment:
This is the actual fix for the initial issue, only removing this filtering which prevented selecting these addresses even though they are used in orders/carts